bin/strings-import.js

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env node
+"use strict";
+
+/* eslint-disable no-console */
+
+/* globals cd, cp, ls, mkdir, rm */
+require("shelljs/global");
+
+// Ues the first script argument as the source of localized strings
+const [, , stringsPath] = process.argv;
+const stringsRepo = require("simple-git")(stringsPath);
+
+// Update strings repository to the latest version
+function pullStrings() {
+  return new Promise((resolve, reject) => {
+    console.log(`Updating L10n repo: ${stringsPath}`);
+
+    // Get the strings revision
+    stringsRepo.pull().show(["--format=%H", "--no-patch"], (err, val) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+      resolve(val.trim());
+    });
+  });
+}
+
+// Replace and update each locale's strings
+function updateLocales(revision) {
+  return new Promise(resolve => {
+    cd("locales");
+
+    console.log("switching to and deleting existing l10n tree under: locales");
+    ls().forEach(dir => {
+      // Keep en-US as it might have newer strings
+      if (dir !== "en-US") {
```

Comment:
Moved to `DEFAULT_LOCALE` although we should be careful if we switch to "en" as mozilla-central expects "en-US"